fronts-client/src/actions/Cards.ts

```diff
@@ -222,6 +225,7 @@ const mayLowerCardBoostLevelForDestinationGroup = (
 		const { collection } = selectGroupCollection(state, groupId);
 		const group = selectGroups(state)[groupId];
 		if (collection?.type === FLEXIBLE_GENERAL_NAME) {
+			// if we move a gigaboosted card to a standard group, we set megaboost
 			if (
```

Comment:
We could lift up an early return here to say 
`if (!group || group.id === null)` {
return
}`

this would allow us to remove this repetition form lines 229, 244, and 254